docs/guides/ultimate-guides/torrents/ultimate-intro-to-torrents/index.md

```diff
@@ -0,0 +1,152 @@
+
+![DHT Request](./img/dht-diagram-dark-mode.svg#gh-dark-mode-only)![DHT Request](./img/dht-diagram-light-mode.svg#gh-light-mode-only)
+
+### Leechers and Seeders
+
+BitTorrent is built around the idea of users sharing files with each other, the system only works if users are willing to share the files they have downloaded with other users.
+
+A "leecher" is a name typically given to users who download files and immediately stop sharing them. A "leech" is someone who takes from the BitTorrent network without giving back.
+
+If you are still actively downloading a file and simultaneously uploading to other peers, you are commonly referred to as a "peer". A peer generically refers to any user in a swarm, but is often used to refer to users who are still downloading the file.
+
+Finally, a "seeder" is a user who has completed downloading a file and continues to share the file with other users. Because of the decentralized nature of BitTorrent and the concept of "pieces", it is extremely beneficial to have seeders in the swam who have full copies of the file. Without enough seeders in a swarm, it is common for all active members in a swarm to exchange pieces with each other, but if no user has the complete file, it is possible that no user will ever complete the download.
+
+### Private Trackers
+
+A private tracker refers to a Tracker, usually backed by a website or community, that requires registration to login and access. When downloading torrents from a private torrent tracker website, the tracker URL is usually unique to your account and is used to track your ratio of upload to download. While limiting the amount of users who can join may appear counter-intuitive to the decentralized nature of BitTorrent, private trackers often mitigate a small portion of privacy concerns by ensuring non-community members are not able to spy on the activity in the swarm.
+
+It may go without saying that private trackers are usually closed to registration, requiring an invite from an existing member to join, or some trackers will open registration for a short period of time to allow new members to join.
+
+In private tracker environments, it may be that your tracker requires you to maintain a certain ratio of upload to download. It is usually the case that you must at least upload as much as you download, or otherwise face some penalty, such as being banned from the tracker.
```

Comment:
[Proofreading]
```suggestion
In private tracker environments, it may be that your tracker requires you to maintain a certain ratio of upload to download. Typically, you are required to at least upload as much as you download. Otherwise, you face a penalty, such as being banned from the tracker.
```

Reply:
Added a word `you will face`